index.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const core = require("@actions/core");
+const github = require("@actions/github");
+const { Octokit, App } = require("octokit");
```

Comment:
Once ESM mode is enabled this could be refactored to:
```suggestion
import core from "@actions/core";
import github from "@actions/github";
import { Octokit, App } from "octokit";
```